02-electric/electric-section-one/electric-section-one.html

```diff
@@ -1,41 +1,86 @@
-      t is a long established fact that a reader will be distracted by the
-      readable content of a page when looking at its layout. The point of
-      using Lorem Ipsum is that it has a more-or-less normal distribution of
-      letters, as opposed to using 'Content here, content here', making it
-      look like readable English. Many desktop publishing packages and web
-      page editors now use Lorem Ipsum as their default model text, and a
-      search for 'lorem ipsum' will uncover many web sites still in their
-      infancy. Various versions have evolved over the years, sometimes by
-      accident, sometimes on purpose (injected humour and the like).
-      </p>
-      <button class="electric-section-one-btn">button</button>
-   </section>
-  </section>
-  
-</body>
-</html>
\ No newline at end of file
+      </div> -->
+      <h1 class="electric-section-one-title">Washing Machines</h1>
+      <section class="electric-section-one-subcontainer">
+        <div class="electric-section-one-part-one-img-container">
+          <img
+            class="electric-section-one-part-one-img"
+            src="../../00-overall/05-images/placeholder.svg"
+            alt="image section two"
+          />
+        </div>
+
+        <h1 class="electric-section-one-part-one-title">
+          Washing clothes at a lower temperature could save you up to £40 a year
+        </h1>
+
+        <p class="electric-section-one-part-one-content">
+          Modern washing machines can clean clothes effectively at lower
+          temperatures. Changing from 40°C to 30°C means you could get 3 cycles
+          instead of 2 using the same amount of energy, depending on your
+          washing machine.
+        </p>
+        <div class="electric-section-one-part-two-img-container">
+          <img
+            class="electric-section-one-part-two-img"
+            src="../../00-overall/05-images/placeholder.svg"
+            alt="image section two"
+          />
+        </div>
+
+        <h1 class="electric-section-one-part-two-title">
+          Always fully load the washing machine could you save up t0 £11 a year
+        </h1>
+
+        <p class="electric-section-one-part-two-content">
+          Make sure everyone has put their dirty clothes in the washing machine
+          so its full before it starts a cycle
+        </p>
+        <div class="electric-section-one-part-two-img-container">
+          <img
+            class="electric-section-one-part-two-img"
+            src="../../00-overall/05-images/placeholder.svg"
+            alt="image section two"
+          />
+        </div>
+
+        <h1 class="electric-section-one-part-three-title">
+          Air dry laundry when it is possible
+        </h1>
+        <h4 class="electric-section-one-part-three-subtitle">
+          Using your tumble dryer less could save you £70 a year
+        </h4>
+
+        <p class="electric-section-one-part-three-content">
```

Comment:
make this -content to -paragraph
because content is a very broad term in english

Reply:
done